package.json

```diff
@@ -17,9 +17,10 @@
     }
   ],
   "scripts": {
-    "build": "grunt clean:all && grunt build",
+    "build": "grunt clean:all && grunt build && yarn build:docs",
     "build:server": "grunt clean:server && grunt build:server",
     "build:client": "grunt clean:client && grunt build:client",
+    "build:docs": "cd api && swag init -g ./http/handler/handler.go",
```

Comment:
can we do "swag init -g ./api/http/handler/handler.go" directly?

Reply:
no, have to use swaggo inside a go package